### PR TITLE
Bdog 1992

### PR DESCRIPTION
**Pull request by peteanning**
Fixed issue with only hiding config column headers

### CODE REVIEW

File: app/views/ServiceConfigPage.scala.html [hunk 1]
```diff
@@ -29,9 +29,11 @@
   request: RequestHeader
 )
 
-@display(env: String, textIfTrue: String, textIfFalse: String = "") = {
-    @defining(deployments.map(_.environment.asString) :+ "local"){deployedEnvs =>
-        @if(!deployedEnvs.contains(env)){ @textIfTrue } else { @textIfFalse }
+@hasDeployment(env: ConfigEnvironment, textIfTrue: String, textIfFalse: String = "") = @{
+    if ((deployments.map(_.environment.asString) :+ "local").contains(env.asString)) {
```
Comment (colin-lamed) [inline app/views/ServiceConfigPage.scala.html:33]: Could avoid comparing the `asString` representation with `(deployments.map(ConfigEnvironment.ForEnvironment.apply) :+ ConfigEnvironment.Local).contains(env)`

Comment (colin-lamed) [reply app/views/ServiceConfigPage.scala.html:33]: Providing both `textIfTrue` and `textIfFalse` runs the risk of being mixed up, at least harder to follow (note, it's being called without named params).
I think using `@defining(..){deployments =>` which would move the boolean logic to the call points could be simpler? e.g. `@if(deployments.contains(env) { "hidden"}`

Comment (peteanning) [reply app/views/ServiceConfigPage.scala.html:33]: Added named parameters as discussed

Comment (peteanning) [reply app/views/ServiceConfigPage.scala.html]: Refactored to compare types not Strings

**Review by colin-lamed — APPROVED** [review]

**Pull request merged**